### PR TITLE
Add support for an array of objects as input

### LINKED ISSUE #13: Weird behaviour with array of objects as input
```
camelcaseKeys([{'foo-bar': true}], {deep:true})
```
gives
```
{
    0: {
        fooBar: true
    }
}
```
I was expecting 
```
[
    {
        fooBar: true
    }
]
```
Since
```
camelcaseKeys({'test': [{'foo-bar': [{'foo-baz': true}]}]}, {deep:true});
```
Gives
```
{
    test: [
        {
            fooBar: [{ fooBaz }]
        }
    ]
}
```

### PR DESCRIPTION
**Pull request by duynguyenhoang**
Fixes #13

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "camelcase-keys",
-  "version": "4.1.0",
+  "version": "4.1.1",
```
Comment (sindresorhus) [inline package.json]: Don't bump version in a pull request. It's up to the maintainers when, how, and what version to bump.

File: package.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
     "quick-lru": "^1.0.0"
   },
   "devDependencies": {
-    "ava": "*",
+    "ava": "^0.23.0",
```
Comment (sindresorhus) [inline package.json]: Don't do unrelated changes

File: test.js [hunk 3]
```diff
@@ -17,3 +17,11 @@ test('deep option', t => {
 		{fooBar: true, obj: {oneTwo: false, arr: [{threeFour: true}]}}
 	);
 });
+
+test('array convert', t => {
```
Comment (sindresorhus) [inline test.js]: `array convert` => `accepts an array of objects`

**Comment by sindresorhus** [comment]
Needs to be documented.

**Comment by duynguyenhoang** [comment]
@sindresorhus Please check my PR again. Thanks

**Comment by sindresorhus** [comment]
Thanks :)
